Source/Core/Core/Core.cpp

```diff
@@ -32,11 +32,16 @@
 #include "Common/Timer.h"
 
 #include "Core/Analytics.h"
-#include "Core/Boot/Boot.h"
 #include "Core/BootManager.h"
 #include "Core/ConfigManager.h"
 #include "Core/CoreTiming.h"
 #include "Core/DSPEmulator.h"
+#include "Core/Host.h"
+#include "Core/MemTools.h"
+#ifdef USE_MEMORYWATCHER
```

Comment:
We should move this to the end of the includes, so it stops mucking up the sorting.

Reply:
Done.